src/main/webapp/zoom.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="styles/techstyle.css">
+  <script src="script.js"></script>
+  <script src="scripts/login-script.js" defer></script>
+</head>
+<body>
+  <div id="nav-placeholder"> </div>
+
+  <div class="content">
+    <div class="header">
+      <h1>Zoom</h1>
+      <p>Description of Zoom</p>
+      <a href="https://zoom.us/"></a>
+    </div>
+
+    <div id = "helpful-links">
```

Comment:
Styling: Spaces around the equal sign.  Not quite sure where the styling reference is from but we discussed this during our stand up meeting.

Reply:
Fixed.